upgrade-notes/backend_20250131_220504.md

```diff
@@ -5,6 +5,9 @@
 - all your custom icons, rendered as `<i class="svg svg-{iconName}"></i>`, should be replaced with the `ux_icons` function
 - you can use upgrade command `bin/console upgrade:replace-obsolete-icons` to replace all custom icons in your project in twig templates
     - check if all icons were replaced correctly and are displayed as expected (search for `svg-` in your codebase)
-- icons in JavaScript files should be replaced manually, see the documentation page [Icons](https://docs.shopsys.com/administration/icons) for more information
+    - your custom icons may be used referenced via alias or directly by their name. It depends on your preferences
+    - remember to run `php bin/console ux:icons:lock` to download any custom icons you are using and commit them
+    - see the documentation page [Icons](https://docs.shopsys.com/administration/icons#1-using-icons-in-twig-templates) for more information
+- icons in JavaScript files should be replaced manually, see the documentation page [Icons](https://docs.shopsys.com/administration/icons#2-using-icons-in-javascript) for more information
```

Comment:
please, update the line in the docs regarding importing the icons in Javascript:
```diff
- import InfoCircle from 'icons/info-circle-filled.svg';
+ import InfoCircle from 'icons/tabler/info-circle-filled.svg';
```

Reply:
fixed, thanks ca58a146b6